examples/bread.json

```diff
@@ -31,7 +31,7 @@
         {
             "type": "linear-equality",
             "names": ["flour", "water", "salt", "yeast"],
-            "rhs": 1
+            "rhs": 1000
```

Comment:
no change required

Reply:
Sure? Because I think otherwise the provided data 
`
          "data": [
          [700, 250, 25, 25, "A", 2, 1, 20, 90, 45, 180, 4, null, 3, 3],
          [800, 150, 25, 25, "B", 4, 2, 30, 90, 45, 180, 1, 1500, 5, 2],
          [900,  50, 25, 25, "C", 2, 3, 20, 90, 45, 180, 3, 150, null, 5]
`
do not fit (the first four entries add up to 1000). Or should the data be scaled accordingly instead of the rhs?